src/routes/tickets/util/getMetadataFromTicket.ts

```diff
@@ -1,11 +1,29 @@
 import { ITicket, ITicketMetaData } from '../controllers/ticketController';
 
+
 export const getMetadataFromTicket = (data: ITicket[]): ITicketMetaData[] => {
-	const metadata = data;
+  const metadata = data;
+  let flattenedMessage = '';
+
+  // Take all the messages put them together into a single string and if the string is equals or grater than 50 return. Once 50 add three dots at the end of te string and return that as previewString
+
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
+  metadata[0].messages[0].body.content.forEach(message => {
+    message.content?.forEach(content => {
+      if (flattenedMessage.length >= 50) return;
+      if (content.type === 'text' && typeof content.text === 'string') {
+        flattenedMessage = `${flattenedMessage} ${content.text}`;
+      }
+    });
+  });
 
-	metadata[0].previewText = metadata[0].messages[0].text.slice(0, 50);
+  const newMessageArr = flattenedMessage.slice(0, 50).split('');
+  newMessageArr.push('...');
+  metadata[0].previewText = newMessageArr.join('');
```

Comment:
I think a simpler version might be just 
```js
metadata[0].previewText = flattenedMessage.slice(0, 50) + "..."
```

Reply:
@binaryoverload Can you make this a suggestion like the one above regarding the flatmap?